docs/REST API/rest-api.md

```diff
@@ -7528,6 +7531,118 @@ Returns information about the specified software version.
+| order_key               | string  | query | What to order results by. Allowed fields are: `cve`, `cvss_score`, `epss_probability`, `cve_published`, `created_at`, and `host_count`. Default is `created_at` (descending).      |
+| order_direction | string | query | **Requires `order_key`**. The direction of the order given the order key. Options include `asc` and `desc`. Default is `asc`. |
+
+
+##### Default response
+
+`Status: 200`
+
+```json
+{
+  "vulnerabilities": [
+    {
+      "cve": "CVE-2022-30190",
+      "created_at": "2022-06-01T00:15:00Z",
+      "host_count": 1234,
+      "host_count_updated_at": "2023-12-20T15:23:57Z",
+      "details_link": "https://nvd.nist.gov/vuln/detail/CVE-2022-30190",
+      "cvss_score": 7.8,// Available in Fleet Premium
+      "epss_probability": 0.9729,// Available in Fleet Premium
+      "cisa_known_exploit": false,// Available in Fleet Premium
+      "cve_published": "2022-06-01T00:15:00Z",// Available in Fleet Premium
+      "cve_description": "Microsoft Windows Support Diagnostic Tool (MSDT) Remote Code Execution Vulnerability.",// Available in Fleet Premium
+      "resolved_in_version": ""// Available in Fleet Premium
+    }
+  ],
+  "count": 123,
+  "counts_updated_at": "2024-02-02T16:40:37Z",
+  "meta": {
+    "has_next_results": false,
+    "has_previous_results": false
+  }
+}
+```
+
+
+### Get vulnerability
+
+Retrieve details about a vulnerability and its affected software and OS versions.
+
+#### Parameters
+
+| Name     | Type     | In    | Description                                                                                     |
+| ---      | ---      | ---   | ---                                                                                             |
+| cve      | string  | path | The cve to get information about (including "cve-" prefix, case-insensitive).                       |
+
+`GET /api/v1/fleet/vulnerabilities/:cve`
+
+#### Example
+
+`GET /api/v1/fleet/vulnerabilities/cve-2022-30190`
+
+##### Default response
+
+`Status: 200`
+
+```json
+"vulnerability": {
+  "cve": "CVE-2022-30190",
+  "created_at": "2022-06-01T00:15:00Z",
+  "host_count": 1234,
+  "host_count_updated_at": "2023-12-20T15:23:57Z",
+  "details_link": "https://nvd.nist.gov/vuln/detail/CVE-2022-30190",
+  "cvss_score": 7.8,// Available in Fleet Premium
+  "epss_probability": 0.9729,// Available in Fleet Premium
+  "cisa_known_exploit": false,// Available in Fleet Premium
+  "cve_published": "2022-06-01T00:15:00Z",// Available in Fleet Premium
+  "cve_description": "Microsoft Windows Support Diagnostic Tool (MSDT) Remote Code Execution Vulnerability.",// Available in Fleet Premium
+  "resolved_in_version": "", // Available in Fleet Premium
+  "os_versions" : [
+    {
+      "os_version_id": 5,
+      "host_count": 5,
+      "name": "Microsoft Windows 11 Enterprise 22H2 10.0.22621.2715",
+      "name_only": "Microsoft Windows 11 Enterprise 22H2",
```

Comment:
@rachaelshaw the ["name" column in Figma](https://www.figma.com/file/iWRGEtryWkdPru8E9sJRiK/%2315919-Add-Vulnerabilities-page?type=design&node-id=336%3A5740&mode=design&t=MBvYnOLpxxgso3Pj-1) has 3 x "macOS" as examples – that column should actually look more like this image, a platform and a version, and less like just a platform ("macOS", "Windows") yea? This column should be taken from this `name_only` field right?

<img width="333" alt="Screenshot 2024-02-06 at 5 52 04 PM" src="https://github.com/fleetdm/fleet/assets/61553566/545abb15-6b48-47f3-b67f-1c5db1d76e76">